example/App.js

```diff
@@ -16,45 +21,113 @@ export default class AppContainer extends React.Component {
 
 @connectActionSheet
 class App extends React.Component {
+  state = {
+    selectedIndex: null,
+  };
+
+  _updateSelectionText = (selectedIndex: number) => {
+    this.setState({ selectedIndex });
+  };
+
+  _renderButton = (title: string, onPress: () => void) => (
+    <View style={{ marginBottom: 12 }}>
+      <Entypo.Button
+        name="code"
+        backgroundColor="#3e3e3e"
+        onPress={onPress}>
+        <Text style={{ fontSize: 15, color: '#fff' }}>{title}</Text>
+      </Entypo.Button>
+    </View>
+  );
+
+  _renderSelectionText = () => {
+    const { selectedIndex } = this.state;
+    const text = selectedIndex === null
+      ? 'No Option Selected'
+      : `Option #${selectedIndex + 1} Selected`;
+
+    return (
+      <View style={{ marginTop: 20 }}>
+        <Text style={{ textAlign: 'center', color: 'blue', fontSize: 16 }}>
+          {text}
+        </Text>
+      </View>
+    )
+  };
+
   render() {
     return (
-      <View style={styles.container}>
-        <View style={{ marginBottom: 30 }}>
-          <Text style={{ textAlign: 'center' }}>
-            Hello! This is a simple example app to demonstrate @expo/react-native-action-sheet.
+      <View style={{ flex: 1 }}>
+        <ScrollView style={styles.scrollView} contentContainerStyle={styles.contentContainer}>
+          <View style={{ marginBottom: 30 }}>
+            <Text style={{ textAlign: 'center', fontSize: 16 }}>
+              {"Hello!\n\nThis is a simple example app to demonstrate @expo/react-native-action-sheet."}
+            </Text>
+          </View>
+          {this._renderButton('Options Only', this._onOpenActionSheet(false, false, false, false, false))}
+          {this._renderButton('Title', this._onOpenActionSheet(true, false, false, false, false))}
+          {this._renderButton('Title & Message', this._onOpenActionSheet(true, true, false, false, false))}
+          {this._renderButton('Icons', this._onOpenActionSheet(false, false, true, false, false))}
+          {this._renderButton('Title, Message, & Icons', this._onOpenActionSheet(true, true, true, false, false))}
+          {this._renderButton('Use Separators', this._onOpenActionSheet(true, false, true, true, false))}
+          {this._renderButton('Custom Styles', this._onOpenActionSheet(true, true, true, false, true))}
+          {this._renderSelectionText()}
+          <Text style={{ marginTop: 32 }}>
+            Note: Icons and custom text styles are only available on Android. Separators can only be toggled on Android; they always show on iOS.
           </Text>
-        </View>
-
-        <Entypo.Button name="code" backgroundColor="#3e3e3e" onPress={this._onOpenActionSheet}>
```

Comment:
i think its ok to leave the button as is. You can actually apply the `marginBottom` style directly to `Entypo.Button` as well

Reply:
I remember now why I had to wrap this in a view (after doing what you suggested). For some reason, the Entypo button applies the margin inside the button rather than outside. :/

Reply:
For proof, here's what happens when i applied the margin on the Entypo.Button:

![img_0060](https://user-images.githubusercontent.com/5117473/44425775-72fec280-a552-11e8-9797-dc65d39e1ded.PNG)